client.go

```diff
@@ -226,7 +228,7 @@ func (co *Conn) ReadMsg() (*Msg, error) {
 			return m, ErrSecret
 		}
 		// Need to work on the original message p, as that was used to calculate the tsig.
-		err = TsigVerify(p, co.TsigSecret[t.Hdr.Name], co.tsigRequestMAC, false)
+		err = tsigVerifyGSSAPI(p, co.TsigSecret[t.Hdr.Name], co.tsigRequestMAC, false, co.GSSAPI)
```

Comment:
I would keep the old names and just add the new parameter

Reply:
As it's an exported/public function so I didn't want to change the signature and break backwards compatibility. If that's acceptable I can update the original functions.

Reply:
Based on renaming the interface to `TsigGSS` I shortened the new functions to `tsigGenerateGSS` and `tsigVerifyGSS` to match.